application.py

```diff
@@ -30,6 +32,12 @@
 
 aggregator = Aggregator(path)
 
+#connect to kafka producer
+producer = KafkaProducer(bootstrap_servers=['35.222.250.101:9092'],
```

Comment:
Shouldn't the IP be a part of the config

Reply:
encrypted and added to config